package backend.exception.not_found;

public class UserNotFoundException extends NotFoundException {

    protected String getCustomMessage() {
        return "UÅ¼ytkownik nie istnieje";
    }

    public UserNotFoundException(Throwable causedBy) {
        super(causedBy);
    }

    public UserNotFoundException() {
    }
}
